app/src/App.vue

```diff
@@ -47,16 +49,42 @@ onBeforeMount(async () => {
         Sentry.captureException(err);
     }
 });
+
+// Wait 1 second to allow the socket connection to be established before checking the connection status
+setTimeout(() => {
+    watch(
+        isConnected,
+        () => {
+            if (!isConnected.value) {
+                useNotificationStore().addNotification({
+                    id: "offlineBanner",
+                    title: "You are offline",
+                    description:
+                        "Do not worry - you can still use the app and browse through the offline content.",
```

Comment:
Suggestion to make it a bit clearer what type of content could be available:

```suggestion
                        "You can still use the app and browse through offline content, but some content (like videos) might not be available.",
```